test/python/topology/test2_job_config.py

```diff
@@ -17,10 +17,18 @@
 class TestJobConfig(unittest.TestCase):
   def setUp(self):
       Tester.setup_streaming_analytics(self, force_remote_build=True)
+      
+      self.is_v2 = False
+      for creds in self.test_config['topology.service.vcap']['streaming-analytics']:
+        if creds['name'] == self.test_config['topology.service.name']:
+          if 'v2_rest_url' in creds['credentials']:
+            self.is_v2 = True            
 
   def test_UnicodeJobName(self):
      """ Test unicode topo names
      """
+     if self.is_v2:
```

Comment:
This is not true, there's a bug in the service, which needs to be fixed. We should not be changing the test due to a service bug.

Reply:
Is it possible to mark it as a known failure for now, rather than a skip?